app/assets/javascripts/notes.js

```diff
@@ -5,4 +5,12 @@ jQuery(document).on('ready page:load',function() {
      		$('#new_note').submit();
      	}
      })
+
+     $('#new_note #note_text').on('keyup keydown change', function() {
```

Comment:
Why do you need al these three events at the same time?
Try looking for an event that will replace all these.